completions/aptitude

```diff
@@ -94,11 +94,11 @@ _comp_cmd_aptitude()
         _comp_compgen -- -W '"${COMPREPLY[@]}"' \
             ${exclude_flags:+-X "-[$exclude_flags]"}
     else
-        _comp_compgen -- -W 'update upgrade safe-upgrade forget-new clean
-            autoclean install reinstall remove hold unhold purge markauto
-            unmarkauto why why-not dist-upgrade full-upgrade download search
-            show forbid-version changelog keep keep-all build-dep add-user-tag
-            remove-user-tag versions'
+        _comp_compgen -- -W 'add-user-tag autoclean build-dep build-depends
```

Comment:
I don't know if it makes sense to offer both `build-dep` and `build-depends`, they seem to be aliases. We used to have the former, but maybe the latter is enough?

Reply:
They are listed together in the manpage, and there are no mentions to differences between them. Likewise for the online manual mentioned by @akinomyoga. It seems that they are aliases, indeed.

Also this: https://github.com/SamB/aptitude/blob/cb9a4e8776084d636c5ddab7aa30d1f7c064d5b5/src/cmdline/cmdline_do_action.cc#L153-L155

I don't have a preference for either, or for having only one or both. And I don't know how to choose. `aptitude help` shows `build-dep`, so there's some sense in keeping it.

Reply:
I think we should not have `build-dep` only, because with that, `aptitude build-depe<TAB>` will fail. Let's go with `build-depends` only -- for example `aptitude b<TAB>` will Just Work with it, without the useless prompt to choose between the two (if we had both).

Reply:
I haven't been so careful to check the aliases. Let's keep only `build-depends`, the longer and more descriptive one, as instructed in `CONTRIBUTING.md`.

https://github.com/scop/bash-completion/blob/0a13be1f6576ee4977552379e11157073b4e841b/CONTRIBUTING.md?plain=1#L136-L139